src/libraries/Common/src/Internal/Cryptography/ILiteSymmetricCipher.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.Security.Cryptography;
+
+namespace Internal.Cryptography
```

Comment:
I want to get all the stuff in Internal\Cryptography out (it should just be S.S.Cryptography).  So please use System.Security.Cryptography

Reply:
Should the file move elsewhere or just fix the namespace?